build.gradle

```diff
@@ -22,16 +23,21 @@ dependencies {
 	// ***** OAUTH DEPENDENCIES *****
 	implementation("com.slack.api:bolt-jetty:1.45.3")
 	// ***** TEST DEPENDENCIES *****
-	testImplementation('org.junit.jupiter:junit-jupiter-engine:5.13.4')
+	testImplementation('org.junit.jupiter:junit-jupiter-api:5.13.4')
+	testRuntimeOnly('org.junit.jupiter:junit-jupiter-engine:5.13.4')
```

Comment:
🍀 praise: Nice find! I'm not sure of what makes a difference between runtime only test dependencies and test implementations, but it's interesting to find these separate packages.